PathPlanning/ClosedLoopRRTStar/closed_loop_rrt_star_car.py

```diff
@@ -152,14 +155,14 @@ def get_goal_indexes(self):
         return fgoalinds
 
     @staticmethod
-    def collision_check_with_xy(x, y, obstacle_list):
+    def collision_check_with_xy(x, y, obstacle_list, robot_radius):
```

Comment:
This is not your change, but this function can be removed because its parent RRT class has this static method. So, could you remove this function?

Reply:
Since the interface to `collision_check_with_xy` and `check_collision` are different, I needed to create a temporary node `tmp_node` to parse the data correctly to `check_collision`. Does this work for you?

Reply:
Great, I'll consider this resolved then.